spec/AsComponent.coffee

```diff
@@ -0,0 +1,252 @@
+if typeof process isnt 'undefined' and process.execPath and process.execPath.match /node|iojs/
+  chai = require 'chai' unless chai
+  noflo = require '../src/lib/NoFlo.coffee'
+  path = require 'path'
+  root = path.resolve __dirname, '../'
+  urlPrefix = './'
+  isBrowser = false
+else
+  noflo = require 'noflo'
+  root = 'noflo'
+  urlPrefix = '/'
+  isBrowser = true
+
+describe 'asComponent interface', ->
```

Comment:
Need a changelog entry for `asComponent` feature

Reply:
Yep, will do that when I prep the release